app/services/participants/sync_dqt_induction_start_date.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module Participants
+  class SyncDqtInductionStartDate < BaseService
+    FIRST_2021_ACADEMIC_DATE = Date.new(2021, 9, 1)
+    FIRST_2023_REGISTRATION_DATE = Cohort.find_by(start_year: 2023)&.registration_start_date || Date.new(2023, 6, 1)
+
+    def initialize(dqt_induction_start_date, participant_profile)
+      @dqt_induction_start_date = dqt_induction_start_date
+      @participant_profile = participant_profile
+    end
+
+    def call
+      return false unless FeatureFlag.active?(:cohortless_dashboard)
+      return false unless dqt_induction_start_date
+      return update_induction_start_date if mentor? || pre_2021_dqt_induction_start_date? || pre_2023_participant?
+      return cohort_missing unless target_cohort
+
+      update_participant
+    rescue StandardError
+      false
+    end
+
+  private
+
+    attr_reader :dqt_induction_start_date, :participant_profile
+
+    delegate :current_induction_record, :mentor?, to: :participant_profile
+    delegate :cohort, to: :current_induction_record, prefix: :participant
+
+    def amend_cohort
+      @amend_cohort ||= Induction::AmendParticipantCohort.new(participant_profile:,
+                                                              source_cohort_start_year: participant_cohort.start_year,
```

Comment:
Where's the `participant_cohort` defined?

Reply:
In the delegation line above this.

Reply:
Cheers, I didn't see it's prefixed :nerd_face: